app/policies/activity_policy.rb

```diff
@@ -2,8 +2,6 @@ class ActivityPolicy < ApplicationPolicy
   class Scope < Scope
     def resolve
       scope.
-        joins(result: :department).
-        where(departments: { slug: user.department_slugs }).
```

Comment:
This is what limits the activities to being for departments you have access to. This will be hard to do now that we want to show different types of activities. My first thought was we could say `joins(item: :department)` and then ensure assignment, result, and coverage all define a `department` association. However, this won't work because `item` is a polymorphic association and we can't call `joins` with it.

I can't come up with a great way to overcome this right now.... Hmmm....